src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -214,6 +214,22 @@ internal void SetExperimentalFeatures(ConfigScope scope, string featureName, boo
             }
         }
 
+        internal bool IsImplicitWinCompatEnabled()
+        {
+            bool? settingValue = ReadValueFromFile<bool?>(ConfigScope.CurrentUser, "ImplicitWinCompatEnabled", null);
+            if (!settingValue.HasValue)
+            {
+                settingValue = ReadValueFromFile<bool?>(ConfigScope.AllUsers, "ImplicitWinCompatEnabled", null);
```

Comment:
Here, you can set the default value to be `false`, by `ReadValueFromFile<bool?>(ConfigScope.AllUsers, "ImplicitWinCompatEnabled", defaultValue: false);`

Reply:
Updated.